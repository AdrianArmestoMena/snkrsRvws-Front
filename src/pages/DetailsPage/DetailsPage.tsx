import DetailReview from "../../components/DetailReview/DetailReview";
import YourReviewsPageStyle from "../YourReviewsPage/YourReviewsPage.style";

const DetailsPage = (): JSX.Element => {
  return (
    <YourReviewsPageStyle>
      <h2 className="tittle">Your Reviews</h2>
      <DetailReview />
    </YourReviewsPageStyle>
  );
};

export default DetailsPage;
