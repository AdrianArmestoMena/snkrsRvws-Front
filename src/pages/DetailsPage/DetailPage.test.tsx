import TestRenderer from "react-test-renderer";
import Wrapper from "../../test-utils/Wrapper";
import DetailsPage from "./DetailsPage";

describe("Given a detail Page function", () => {
  describe("When it's instantiated", () => {
    test("Then it should match the html from Home page", () => {
      const expectedLoginPage = TestRenderer.create(
        <Wrapper>
          <DetailsPage />
        </Wrapper>
      );

      expect(expectedLoginPage).toMatchSnapshot();
    });
  });
});
