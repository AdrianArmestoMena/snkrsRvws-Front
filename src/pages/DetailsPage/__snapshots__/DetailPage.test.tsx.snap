// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a detail Page function When it's instantiated Then it should match the html from Home page 1`] = `
<main
  className="sc-gsnTZi etYpNr"
>
  <h2
    className="tittle"
  >
    Your Reviews
  </h2>
  <article
    className="sc-bczRLJ hfzbse"
  >
    <div
      className="review__mian-container"
    >
      <h2
        className="review__title"
      >
         
      </h2>
      <span
        className="review__author"
      >
        by 
      </span>
    </div>
    <div
      className="review__image-container"
    >
      <img
        alt="bamba"
        className="review__image"
        src="http://localhost:4000/"
        width={500}
      />
    </div>
    <p
      className="review__review"
    />
    <div
      className="review__buttons-container"
    >
      <button
        className="review__button review__view-button btn btn-primary"
        disabled={false}
        type="button"
      >
        Modify
      </button>
      <button
        className="review__button review__delete-button btn btn-primary"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        Delete
      </button>
    </div>
  </article>
</main>
`;
